contracts/src/tokens/drop/DropStorage.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+// interfaces
+import {IDropFacetBase} from "./IDropFacet.sol";
+
+// libraries
+
+// contracts
+
+library DropStorage {
+  // keccak256(abi.encode(uint256(keccak256("diamond.facets.drop.storage")) - 1)) & ~bytes32(uint256(0xff))
+  bytes32 constant STORAGE_SLOT =
+    0xeda6a1e2ce6f1639b6d3066254ca87a2daf51c4f0ad5038d408bbab6cc2cab00;
+
+  struct Layout {
+    address claimToken;
+    uint256 conditionStartId;
+    uint256 conditionCount;
```

Comment:
```suggestion
    uint48 conditionStartId;
    uint48 conditionCount;
```
`uint48` is large enough for its purpose while small enough to pack into one slot.